src/Tools/Consolonia.Templates/ConsoloniaAppTemplate/MainWindow.axaml

```diff
@@ -0,0 +1,14 @@
+<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d" 
+        x:Class="ConsoloniaAppTemplate.MainWindow"
+        Title="Consolonia">
+    <StackPanel VerticalAlignment="Center"
+                   HorizontalAlignment="Center">
+        <TextBlock Text="Welcome to Conslonia!" />
```

Comment:
Typo. By the way, can we include this subproject for resharper analysis? Stuff like formatting, typos etc could be caught by resharper

Reply:
@coderabbitai can you fix this?

Reply:
Nice fix! Can you show me a button to apply it? That was the point so I don't need to code myself